File: ignored-test/python-pytest/Addins_Should_Log_Warning_For_Files_Located_In_Root.py
import pytest

class TestAddins:

    @pytest.mark.skip(reason="")
    def test_should_log_warning_for_files_located_in_root(self):
        # Given
        fixture = NuGetAddinContentResolverFixture(framework, runtime)
        fixture.create_clr_assembly("/Working/file.dll")
        fixture.create_clr_assembly("/Working/file2.dll")
        fixture.create_clr_assembly("/Working/file3.dll")
        # When
        fixture.get_files()
        # Then
        entries = [
            x for x in fixture.log.entries
            if x.level == LogLevel.Warning and 
               x.message == (f"Could not find any assemblies compatible with {framework} in NuGet package {fixture.package.package}. "
                             "Falling back to using root folder of NuGet package.")
        ]
        assert len(entries) == 1

File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/Addins_Should_Log_Warning_For_Files_Located_In_Root.cs
using Xunit;

namespace Cake.NuGet.Tests(net6.0)
{
    public class Addins
    {
        [Fact(Skip = "")]
        public void Should_Log_Warning_For_Files_Located_In_Root()
        {
            {
                // Given
                var fixture = new NuGetAddinContentResolverFixture(framework, runtime);
                fixture.CreateCLRAssembly("/Working/file.dll");
                fixture.CreateCLRAssembly("/Working/file2.dll");
                fixture.CreateCLRAssembly("/Working/file3.dll");
                // When
                fixture.GetFiles();
                // Then
                var entries = fixture.Log.Entries.Where(x => x.Level == LogLevel.Warning && x.Message.Equals($"Could not find any assemblies compatible with {framework} in NuGet package {fixture.Package.Package}. " + "Falling back to using root folder of NuGet package.")).ToList();
                Assert.Single(entries);
            }
        }
    }
}
